firstProject/settings/dev.py

```diff
@@ -16,7 +16,8 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 INSTALLED_APPS = INSTALLED_APPS + [
-    'debug_toolbar'
+    'debug_toolbar',
+    'django_extensions',
```

Comment:
this just adds the django_extensions library so you can use the interactive shell commands:

pip install django_extensions
add them to INSTALLED_APPS
then run
python manage.py shell_plus

from there you can invoke any model without having to load it first. So to invoke objects for BlogDetailPage:

posts = BlogDetailPage.objects.live()
posts = BlogDetailPage.objects.all()
posts[0]